parachain-template/runtime/src/lib.rs

```diff
@@ -752,15 +764,15 @@ impl_runtime_apis! {
 		) {
 			use frame_benchmarking::{list_benchmark, Benchmarking, BenchmarkList};
 			use frame_support::traits::StorageInfoTrait;
-			use frame_system_benchmarking::Pallet as SystemBench;
-			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
+			// use frame_system_benchmarking::Pallet as SystemBench;
```

Comment:
@librelois I was unable to make these benchmarks work with `--features runtime-benchmarks`
`SystemBench`, `CollatorSelection` complains of unable to guess types
`cumulus_pallet_session_benchmarking` requires `pallet_session::Config` to be implemented which leads to other errors.

Any help is appreciated